--- skype-1.6.2/usr/share/applications/skypeforlinux.desktop~	2017-05-29 16:44:16.000000000 +0300
+++ skype-1.6.2/usr/share/applications/skypeforlinux.desktop	2017-06-09 16:19:24.351182157 +0300
@@ -13,7 +13,7 @@
 StartupNotify=true
 StartupWMClass=Skype
 Encoding=UTF-8
-Categories=Network;Application;
+Categories=Network;InstantMessaging;Telephony;
 MimeType=x-scheme-handler/skype;
 X-KDE-Protocols=skype
 Actions=QuitSkype;
