--- skype-4.1.0.20/skype.desktop	2012-11-27 15:54:58.605967486 +0200
+++ skype-4.1.0.20/skype.desktop	2012-11-27 15:55:50.228363600 +0200
@@ -1,11 +1,12 @@
 [Desktop Entry]
 Name=Skype
-Comment=Skype Internet Telephony
+Comment=p2p VoIP application
+Comment[pl]=Aplikacja VoIP p2p
 Exec=env PULSE_LATENCY_MSEC=60 skype %U
-Icon=skype.png
+Icon=skype
 Terminal=false
 Type=Application
 Encoding=UTF-8
-Categories=Network;Application;
+Categories=Qt;Network;InstantMessaging;Telephony;
 MimeType=x-scheme-handler/skype;
 X-KDE-Protocols=skype
